pkg/apis/componentconfig/register.go

```diff
@@ -48,6 +48,7 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&KubeProxyConfiguration{},
 		&KubeSchedulerConfiguration{},
 		&KubeletConfiguration{},
+		&ApiServerConfiguration{},
```

Comment:
Shouldn't this be called `KubeApiServerConfiguration` to follow naming convention for scheduler etc?